test/generate-async.test.ts

```diff
@@ -24,21 +24,21 @@ describe('generate-async', () => {
     expect(order).eql([1, 2])
   })
 
-  test('preflight-first', async () => {
+  test('preflight at the end', async () => {
     const order: number[] = []
     const uno = createGenerator({
       rules: [
         [/^rule$/, () => new Promise(resolve => setTimeout(() => {
-          order.push(2)
+          order.push(1)
           resolve('/* rule */')
-        }, 20))],
+        }, 1000))],
```

Comment:
Do we need the timeout to be this long?

Reply:
I just wanted to make it look like it was at the end.